binary_search.py

```diff
@@ -0,0 +1,36 @@
+"""
+This file implements a binary search algorithm.
+"""
+
+def binary_search(lst, key):
```

Comment:
mi6: Give meaningful variable names

Reply:
Working on it...

*This is an automated message generated by Mi6*